tests/ansible_galaxy/test_collection_info.py

```diff
@@ -62,3 +65,107 @@ def test_save(tmpdir):
             read_fd.seek(0)
             buf = read_fd.read()
             log.debug('buf: %s', buf)
+
+
+def test_parse_error(tmpdir):
+    test_data = {
+        'name': 'foo.foo',
+        'authors': ['chouseknecht'],
+        'license': 'GPL-3.0-or-later',
+        'version': '0.0.1',
+        'description': 'unit testing thing',
+        'foo': 'foo',
+    }
+    file_name = "example_collection_info1.yml"
+    temp_dir = tmpdir.mkdir('mazer_collectio_info_unit_test')
+    temp_file = temp_dir.join(file_name).strpath
+    with open(temp_file, 'w') as fd:
+        yaml.dump(test_data, fd)
+    with pytest.raises(exceptions.GalaxyClientError):
+        with open(temp_file, 'r') as fd:
+            collection_info.load(fd)
+
+
+def test_license_error():
+    test_data = {
+        'name': 'foo.foo',
+        'authors': ['chouseknecht'],
+        'license': 'GPLv2',
+        'version': '0.0.1',
+        'description': 'unit testing thing',
+    }
+    try:
+        CollectionInfo(**test_data)
```

Comment:
These tests should probably live in test/ansible_galaxy/models/test_collection_info_model.py